include/RAJA/exec-cuda/raja_cuda.hxx

```diff
@@ -65,6 +65,12 @@
 namespace RAJA
 {
 
+#if defined(RAJA_ENABLE_CLANG_CUDA)
```

Comment:
We could actually probably remove the macro usage from the code with compiler defines, but we still need it in the CMake.